### PR TITLE
Added UT for TestTimeBase tests

### PR DESCRIPTION
**Pull request by Akanksha-kedia**
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.facebook.presto.type.TestTimeBase
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.986 s - in com.facebook.presto.type.TestTimeBase
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  39.234 s
[INFO] Finished at: 2023-10-22T18:30:51+05:30
[INFO] ------------------------------------------------------------------------

Description 

The newly added unit tests are designed to verify the correct functionality of the TestTimeBase

Added UT for 
- testDistinctFrom()
- testLessThanOrEquals()
- testGreaterThanOrEquals()
- testAddIntervalDayToSecond()
-testSubstractIntervalDayToSecond()

Test plan:

Run the entire test suite to ensure that all tests are passing.

### CODE REVIEW

**Comment by Akanksha-kedia** [comment]
circleci pipeline rerun

**Comment by Akanksha-kedia** [comment]
@tdcmeehan please review

**Comment by jaystarshot** [comment]
Can you update the desciption as to what tests were missing?

**Comment by Akanksha-kedia** [comment]
> Can you update the desciption as to what tests were missing?

I have added the required information do check its wat you are looking for or anything else apart from this.

File: presto-main/src/test/java/com/facebook/presto/type/TestTimeBase.java [hunk 1]
```diff
@@ -214,4 +214,82 @@ public void testIndeterminate()
...
+        assertFunction("TIME '12:34:56.1' <= TIME '12:34:56.0'", BOOLEAN, false);
+        assertFunction("TIME '12:34:56.12' <= TIME '12:34:56.11'", BOOLEAN, false);
+        assertFunction("TIME '12:34:56.123' <= TIME '12:34:56.122'", BOOLEAN, false);
+    }
+
+    @Test
+    public void testGreaterThanOrEquals()
+    {
+        // equality
+        assertFunction("TIME '12:34:56' >= TIME '12:34:56'", BOOLEAN, true);
```
Comment (jaystarshot) [inline presto-main/src/test/java/com/facebook/presto/type/TestTimeBase.java:251]: Regarding Greater than, less than etc Most of these test cases are already tested in the same file above

**Comment by Akanksha-kedia** [comment]
Greater than and equal is also dere

**Comment by jaystarshot** [comment]
There are tests for greater than and equal there too

File: presto-main/src/test/java/com/facebook/presto/type/TestTimeBase.java (hunk 1 above)
Comment (Akanksha-kedia) [reply presto-main/src/test/java/com/facebook/presto/type/TestTimeBase.java:251]: its tested for timestamp and in that we dnt have the same api, please confirm

**Pull request closed without merging**